src/pages/product/ProductDetailInfo/ProductDetailInfo.jsx

```diff
@@ -1,13 +1,57 @@
 import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import ContentsTab from '../Components/ContentsTab';
+import ProductIntro from './ProductIntro';
+import NoteInfo from './NoteInfo';
+import Keyword from './Keyword';
 import './ProductDetailInfo.scss';
 
 class ProductDetailInfo extends Component {
+  constructor() {
+    super();
+    this.state = {
+      productData: {},
+    };
+  }
+
+  getProductData = async () => {
+    const {
+      match: {
+        params: { id },
+      },
+    } = this.props;
```

Comment:
가독성이 좋지 않아서 아래처럼 해주세요!
```suggestion
  const { id } = this.props.match.params;
```

Reply:
감사합니다. 수정했습니다!

https://github.com/wecode-bootcamp-korea/fullstack1-1st-BaseNote-frontend/pull/3/commits/0a8f379491c2f32b7a0618ace9b26587fff73bfa